src/neki.jl

```diff
@@ -1,28 +1,28 @@
 """
-    neki_iter(prob::SolusProblem, θs, fθs)
+    neki_iter(prob::CESProblem, θs, fθs)
```

Comment:
```suggestion
    eki_iter(prob::CESProblem, θs, fθs)
```
Change `neki` -> `eki` throughout, including in file name.